Source/Core/DolphinQt/Settings/AudioPane.cpp

```diff
@@ -46,29 +48,28 @@ AudioPane::AudioPane()
 
 void AudioPane::CreateWidgets()
 {
-  auto* dsp_box = new QGroupBox(tr("DSP Emulation Engine"));
-  auto* dsp_layout = new QVBoxLayout;
+  auto* dsp_box = new QGroupBox(tr("DSP Options"));
+  auto* dsp_layout = new QHBoxLayout;
 
   dsp_box->setLayout(dsp_layout);
-  m_dsp_hle = new QRadioButton(tr("DSP HLE (recommended)"));
-  m_dsp_lle = new QRadioButton(tr("DSP LLE Recompiler (slow)"));
-  m_dsp_interpreter = new QRadioButton(tr("DSP LLE Interpreter (very slow)"));
+  QLabel* dsp_combo_label = new QLabel(tr("DSP Emulation Engine:"));
+  m_dsp_combo = new ConfigComplexChoice(Config::MAIN_DSP_HLE, Config::MAIN_DSP_JIT);
+  m_dsp_combo->Add(tr("HLE (recommended)"), true, true);
+  m_dsp_combo->Add(tr("LLE Recompiler (slow)"), false, true);
+  m_dsp_combo->Add(tr("LLE Interpreter (very slow)"), false, false);
+  m_dsp_combo->Refresh();
```

Comment:
When I first tested this PR, none of them were selected:
![image](https://github.com/user-attachments/assets/854555ea-1b84-4b78-b656-58e68e847a80)

I don't recall having changed my audio config beside switching between HLE/LLE depending on the game I start. I suppose there might/will be users with the same issue. One way to reproduce it, is to set `MAIN_DSP_HLE = True` and `Config::MAIN_DSP_JIT = False`, for instance:
```INI
[DSP]
WASAPIDevice = 
EnableJIT = False
[Core]
DSPHLE = True
```

Reply:
I added a new commit to set a default index if there is no valid one, which fixes this (as there is only one invalid state).